src/Livewire/Servers.php

```diff
@@ -59,4 +70,22 @@ public function placeholder(): Renderable
     {
         return View::make('pulse::components.servers-placeholder', ['cols' => $this->cols, 'rows' => $this->rows, 'class' => $this->class]);
     }
+
+    /**
+     * Determine if the system should be ignored.
+     *
+     * @param  object{ timestamp: int, key: string, value: string }  $system
+     */
+    protected function ignoreSystem(object $system): bool
+    {
+        if ($this->ignoreAfter === null) {
+            return false;
+        }
+
+        $ignoreAfter = is_numeric($this->ignoreAfter)
+            ? (int) $this->ignoreAfter
+            : CarbonInterval::createFromDateString($this->ignoreAfter)->totalSeconds;
+
+        return CarbonImmutable::createFromTimestamp($system->timestamp)->addSeconds($ignoreAfter)->isPast();
+    }
```

Comment:
I've called this `ignoreSystem` to differentiate it from our shared `ignore` method on the `Ignores` trait.